geometry_msgs/PoseArray path
string world_frame
string tcp_frame
string[] names
float64 dt
string underdefinedAxis
float64 interpolation
sensor_msgs/JointState joint_states

---

trajectory_msgs/JointTrajectory plan
float64 computeTime
int64 pointCount
